### PR TITLE
Increment version to 0.16.0

### PR DESCRIPTION
**Pull request by thisac**
Increment version to 0.16.0 for release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-### Bug fixes
+* Return samples from the `QPUDevice` directly instead of resample from the returned results.
+  [(#32)](https://github.com/PennyLaneAI/PennyLane-IonQ/pull/32)
 
 ### Contributors
 
 This release contains contributions from (in alphabetical order):
 
+Dave Bacon, Nathan Killoran
+
```
Comment (antalszava) [inline CHANGELOG.md:12]: :star:

**Review by antalszava — APPROVED** [review]

**Comment by josh146** [comment]
Hey @antalszava @thisac — does this plugin need a release? Looks like nothing has been added since the last release 🙂

**Comment by antalszava** [comment]
Hi @josh146, I believe that the commit from https://github.com/PennyLaneAI/PennyLane-IonQ/pull/32 that changes the way of how QPUs do sampling made it into `master` in PR https://github.com/PennyLaneAI/PennyLane-IonQ/pull/34. That would mean a change in the code, not just the testing, though let me know if I'm missing something! :)

**Comment by josh146** [comment]
Oops, you’re 100% correct @antalszava, I misread the changelog from my notifications!

**Pull request merged**